app/views/products/index.html.erb

```diff
@@ -24,7 +24,7 @@
         <%= hidden_field_tag :authenticity_token, form_authenticity_token -%>
     </div>
     <%= image_tag product.nutriscore_img, class: "nutriscore-img" %>
-        <%= button_tag type: 'submit', class: "button postfix" do %>
+        <%= button_tag type: 'submit', class: "button postfix", data: { disable_with: "" } do %>
```

Comment:
I think we should use some ajax so that when the user clicks the basket button then it automatically changes to a trash icon without needing to refresh